runtime/lua/vim/health.lua

```diff
@@ -407,6 +407,15 @@ function M._check(mods, plugin_names)
   -- Clear the 'Running healthchecks...' message.
   vim.cmd.redraw()
   vim.print('')
+
+  -- Quit with 'q' inside healthcheck buffers.
+  vim.api.nvim_buf_set_keymap(
+    bufnr,
+    'n',
+    'q',
+    '<cmd>bdelete<cr>',
```

Comment:
I think `<c-w>q` is a good convention to continue. No need to close all other windows that are potentially showing this buffer. Plus it's better if `q` behaves consistently where possible.
```suggestion
    '<c-w>q',
```